pom.xml

```diff
@@ -204,8 +203,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>${maven-compiler-plugin.version}</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
Are you wanting to support JDK7 on Scala 2.12? If so, this should flip to `1.7`, if not. The `README` should be updated to state this no longer supports JDK7.